packages/airnode-abi/src/constants.ts

```diff
@@ -0,0 +1,24 @@
+import { ValueOf, TypeTransformation } from './types';
+
+// Upper case letters refer to dynamically sized types, lower case letters refer to statically sized types
+export const PARAMETER_SHORT_TYPES = {
+  B: 'bytes',
+  b: 'bytes32',
+  S: 'string',
+  // Type "string32" is not supported in solidity and we use bytes32 on chain instead. We need to create this
+  // artificially made up type so that users can decide if they want their data encoded as strings or as raw bytes
+  // value.
+  s: 'string32',
+  a: 'address',
+  i: 'int256',
+  u: 'uint256',
+  f: 'bool',
+} as const;
+
+export type ParameterType = ValueOf<typeof PARAMETER_SHORT_TYPES>;
```

Comment:
@andreogle @amarthadan btw. this is a cool trick to create unions of keys and values of an object. This is preferred over enums in Typescript as they are actually poorly designed. See this for more info: https://medium.com/flow-type/typescript-enums-vs-flow-enums-83da2ca4a9b4